infra/nnfw/python/README.md

```diff
@@ -39,17 +39,25 @@ $ twine upload --repository-url https://test.pypi.org/legacy/ dist/*
 You can install the package as follows:
 
 ```
-$ pip install -i https://test.pypi.org/simple/ nnfwapi
+$ pip install -i https://test.pypi.org/simple/ nnfw-onert
 ```
 
 By specifying the version, you can use a specific version of the package. (recommended)
 
 ```
-$ pip install -i https://test.pypi.org/simple/ nnfwapi==0.1.1
+$ pip install -i https://test.pypi.org/simple/ nnfw-onert==0.1.1
 ```
 
 This definition has to be set at the top of the script using nnfw python API.
 
 ```
-from nnfwapi.libnnfw_api_pybind import *
+import nnfw_onert
```

Comment:
```suggestion
from nnfw_onert import onert
```
Wouldn't it be more inline with actual usage? As far as I understand `nnfw_onert` is just a namespace package, so pointing out to shorter path would be even better, just to use `onert.something` later on.